kie-spring/pom.xml

```diff
@@ -258,6 +258,7 @@
           <systemPropertyVariables>
             <kie.maven.settings.custom>${project.build.testOutputDirectory}/kie-custom-settings.xml</kie.maven.settings.custom>
           </systemPropertyVariables>
+          <forkMode>always</forkMode>
```

Comment:
I don't think that's possible using just surefire config.

I agree that `<forkMode>always</forkMode>` is quite heavy. It usually suggests that the tests are not properly isolated or that they are not cleaning-up after themselves.